doc/releases/release-notes-4.0.rst

```diff
@@ -1068,8 +1068,6 @@ Libraries / Subsystems
 
 * Random
 
-* SD
```

Comment:
@decsny and I were just talking in another PR that it may be better to just leave the section in and say "No changes in this release" or something like that.  Thoughts?

Reply:
I was also considering this. I'm fine with that approach- I'll update this PR